src/main/kotlin/org/rust/ide/annotator/RsCargoCheckAnnotator.kt

```diff
@@ -82,9 +82,12 @@ class RsCargoCheckAnnotator : ExternalAnnotator<CargoCheckAnnotationInfo, CargoC
             val primarySpan = message.spans
                 .filter { it.is_primary }
                 .filter { it.isValid() }
-                .filter { filePath.endsWith(PathUtil.getCanonicalPath(it.file_name)) }
```

Comment:
on windows filePath and it.file_name have different kind of slashes, that's why.